### PR TITLE
Fix some Make issues, test project creation, fix it

### PR DESCRIPTION
**Pull request by sbrunner**
replace #1189

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/vars_+package+.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        - mapserver_substitution
+
+    functionalities:
+        # Functionalities that are accessible to anonymous
+        # users.
+        anonymous:
+            print_template:
+            - 1 A4 portrait demo
+            - 2 A3 portrait demo
+            default_basemap: plan
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/create/vars_+package+.yaml]: What about using a more generic name than "plan" here?

File: travis/build-new-peoject.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
```
Comment (asaunier) [inline travis/build-new-peoject.sh]: Typo in the name of the file "build-new-peoject.sh" => "build-new-project.sh"

File: travis/empty-make-new-peoject.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```
Comment (asaunier) [inline travis/empty-make-new-peoject.sh]: Typo in the name of the file "empty-make-new-peoject.sh" => "empty-make-new-project.sh"

File: c2cgeoportal/scaffolds/create/vars_+package+.yaml (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/vars_+package+.yaml]: like ortho ?

Comment (asaunier) [inline c2cgeoportal/scaffolds/create/vars_+package+.yaml]: Well, ortho is a special baselayer. Can we really use it as the default basemap?
I meant something in english, not in french like "plan".

**Pull request merged**